source/loader/windows/lib_init.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ * Copyright (C) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+
+#include "../ur_lib.h"
```

Comment:
Is there a way to avoid relative file paths here?

Reply:
Done